src/seq/fasta.jl

```diff
@@ -0,0 +1,1123 @@
+# FASTA sequence types
```

Comment:
I think it would be nice if generated files contain a comment like "This file is generated from *.rl. Do not edit me!" at the top of them.